### ISSUE TITLE
Resume the training process

### ISSUE THREAD

**Issue by vic85821** [cmlab-mira/template#23]
When the user doesn't modify the `mode` and `target` attributes of the monitor, the process would be forced to stop.
After the early stopping:
- the user would like to resume the training process for more epoches.
- the user would like to change the random seed, the learning rate, or the optimizer and resume the process

**Comment by peter850706** [comment]
For the first situation, I think the user should increase `early_stop` to resume. Have added more messages in 247275c.
I think the second situation is tricky as the `Monitor` can't know this situation (even the `BaseTrainer`), so if you have a solution to it, welcome to reopen this issue!
